audit/projects/k8s-staging-e2e-test-images/iam.json

```diff
@@ -20,10 +20,42 @@
       ],
       "role": "roles/cloudbuild.serviceAgent"
     },
+    {
```

Comment:
agree - should be disabled.

Reply:
given your explanation for "I clicked on console and this happens automatically" this is almost undoubtedly my fault, I navigate these services all the time on a number of projects

added to https://github.com/kubernetes/k8s.io/issues/1675